tests/service/test_person_service.py

```diff
@@ -0,0 +1,49 @@
+import pytest
+
+from mixer.backend.flask import mixer
+
+from nb2.service.dtos import CreatePersonDTO
+from nb2.service.exceptions import EmptyRequiredFieldException
+from nb2.service.person_service import *
+
+
+@pytest.mark.parametrize('num_people', (2, 10))
+def test_get_all_people_returns_all_people(client, session, num_people):
+    # Assert that get_all_people works in the initial, empty state
+    assert len(get_all_people()) == 0
+
+    mixer.cycle(num_people).blend(Person)
+
+    assert len(get_all_people()) == num_people
+
+
+def test_get_person_by_slack_user_id(client, session):
+    person = mixer.blend(Person)
+    retrieved_person = get_person_by_slack_user_id(person.slack_user_id)
+
+    assert person.id == retrieved_person.id
+
+
+def test_create_person(client, session):
+    assert len(Person.query.all()) == 0
```

Comment:
I'm wondering if we should take this as a given. It feels like we shouldn't have to assert this is 0 in multiple tests. We could test our tests by having a single test that asserts the db is empty?

Reply:
Yeah I felt kinda silly writing it. My confidence was just shaken after trying to write the command unit tests, but I guess the sessions are actually working as intended in this case